suites/quincy/rgw/tier-1_rgw_ms_upgrade_5-3_latest_to_6x_latest.yaml

```diff
@@ -27,7 +27,7 @@ tests:
                   service: cephadm
                   args:
                     rhcs-version: 5.3
-                    release: "tier-0"
+                    release: latest
```

Comment:
IMO, it is better to keep tier-0 itself, since as per my understanding latest section in the recipe file will get updated during the listner-job even if container is not available latest will get updated.

since our upgrade suite is for tier-1 it is better to take build from tier-0 section.

Reply:
looks like currently we don't have section "tier-0" in recipe file.